android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -668,6 +670,48 @@ public void setConsentFlowDebugUserGeography(final String userGeography)
         debugUserGeographyToSet = userGeography;
     }
 
+    @ReactMethod
+    public void showCmpForExistingUser(final Promise promise)
+    {
+        if ( sdk == null )
+        {
+            promise.reject( new IllegalStateException( "ERROR: Failed to execute showCmpForExistingUser() - please ensure the AppLovin MAX React Native module has been initialized by calling 'AppLovinMAX.initialize(...);'" ) );
+            return;
+        }
+
+        Activity currentActivity = maybeGetCurrentActivity();
+        if ( currentActivity == null )
+        {
+            promise.reject( new IllegalStateException( "ERROR: Failed to execute showCmpForExistingUser() - unable to get current Activity." ) );
+            return;
+        }
+
+        AppLovinCmpService cmpService = sdk.getCmpService();
+        cmpService.showCmpForExistingUser( currentActivity, (@Nullable final AppLovinCmpError error) -> {
+            if ( error == null )
+            {
+                promise.resolve( null );
+            }
+            else
+            {
+                promise.resolve( error.getCmpCode() );
+            }
```

Comment:
nit our codebase always `return`s early:
```suggestion
            if ( error == null )
            {
                promise.resolve( null );
                return;
            }

            promise.resolve( error.getCmpCode() );
```